tox.ini

```diff
@@ -7,27 +7,24 @@ passenv =
     PYTHON_VERSION
     TOXENV
 setenv =
-    PYTHON_VERSION=3.7
-# TODO: not sure how to test multiple versions in the same invocation
-    TOXENV=check,py27,pypy,py34,py35,py36,py37,py38
+    PYTHON_VERSION=3.8
+    TOXENV=check,pypy,py38,py39,py310,py311,py312
 deps =
 usedevelop = false
 depends =
 skip_install = true
 ignore_errors = true
 commands =
-    docker build --build-arg PYTHON_VERSION=3.8 -t redo-{envname}-py38-test -f Dockerfile.test .
+    docker buildx build --build-arg PYTHON_VERSION=3.8 -t redo-{envname}-py38-test -f Dockerfile.test .
```

Comment:
Please leave that change out of this PR.  (I know it'll break things for me, at least.)